### DISCUSSION TITLE
[REPORTE-PLANETAS] dos planetas con misma cantidad de pedidos

### DISCUSSION THREAD

**Discussion by ManuelV03** [IIC2133-PUC/2024-1#23] [Tarea 0] [answered]
Hola, respecto a la funcionalidad REPORTE-PLANETAS, en el enunciado dice que hay que imprimir los planetas ordenados de mayor a menor según la cantidad de pedidos que se han entregado, mi duda surge debido a que en el enunciado nunca especifican que debería suceder si es que ocurre un "empate" es decir si en dos planetas se han entregado la misma cantidad de pedidos. Quería saber qué criterio se espera que utilicemos para este desempate, en especial para no tener problema con los testcases. De antemano muchas gracias.

**Comment by sachondo7** [comment, marked as answer]
Hola, en caso de empate debieses imprimir primero el planeta con menor ID

**Reply by ManuelV03** [reply]
Gracias!
